stanovy_vpsFree.cz.md

```diff
@@ -154,149 +166,200 @@ informačním systému spolku
 5. Rada Spolku je povinna do programu Schůze zahrnout body, které navrhne
 Kontrolní komise a jakýkoliv člen Spolku
 
-### §11 Jednání
+### §11 Jednání schůze členů
 
 1. Jednání probíhá podle programu, který přítomní členové schválí na začátku
 Schůze
-2. Řádně svolaná Schůze členů je vždy usnášeníschopná. Schůze členů rozhoduje 
-prostou většinou hlasů přítomných členů.
+2. Řádně svolaná Schůze členů je usnášeníschopná při účasti alespoň třetiny
+členů. Schůze členů rozhoduje prostou většinou hlasů přítomných členů.
 3. Zahájení a průběh jednání Schůze členů řídí předseda Rady spolku, nebo jiný
-člen pověřený Radou Spolku
+člen pověřený Radou spolku
 4. O Schůzi členů se pořizuje zápis; na začátku jednání se volí zapisovatel
 a nejméně jeden ověřovatel. Zapisovatel je povinen zápis do 30 dnů po jednání
 rozeslat elektronickou poštou všem členům spolku a vyvěsit jej v informační
 systému. Ověřovatel(é) ověří pravdivost a úplnost zápisu a zajistí případnou
-nápravu nepřesností ještě před rozesláním.
+nápravu nepřesností ještě před rozesláním
 5. Zápis o Schůzi členů musí obsahovat:
   1. název a sídlo Spolku,
   2. místo, datum a dobu konání Schůze členů,
   3. jméno předsedajícího jednání, ověřovatelů a zapisovatele
   4. prezenci přítomných
-  5. popis projednaných bodů programu  6. rozhodnutí Schůze členů s uvedením
-výsledků hlasování
-  7. obsah námitky člena Spolku týkající se rozhodnutí Schůze členů, jestliže
-o to dotyčný žádá
+  5. popis projednaných bodů programu
+  6. rozhodnutí Schůze členů s uvedením výsledků hlasování
+  7. obsah námitky člena Spolku týkající se rozhodnutí Schůze členů, jestliže o
+  to dotyčný žádá
+6. Umožňuje se konání náhradní Schůze členů dle ustanovení § 257 občanského zákoníku.
+
+### §11a Jednání na dálku (per rollam)
+
+1. Jednání Schůze členů dle § 11 může probíhat způsobem umožňujícím dálkovou
+účast; taková schůze může probíhat částečně či zcela ve virtuálním prostoru
+prostřednictvím prostředků komunikace na dálku, a to v informační systému
+(prostředí) Spolku, event. jiným způsobem stanoveným Radou spolku nebo
+příslušným interním řádem
+2. Rada Spolku v pozvánce na Jednání Schůze členů stanoví podmínky dálkové
+účasti na dané Schůzi členů, nebo odkáže na řád pro hlasování Schůze členů na
+dálku (per rollam) přijatý dle § 18 Stanov
+3. Při jednání na dálku mohou účastníci Schůze členů s hlasovacím právem
+hlasovat na dálku; takové hlasování má stejné účinky, jako hlasování v
+přítomnosti
+4. Program Schůze členů, která probíhá částečně či zcela na dálku, se považuje
+za schválený, pokud nedojde k jeho zamítnutí účastníky schůze do 1 hodiny od
```

Comment:
Jestli je to tak, že zamítnout program lze pouze v intervalu <začátek schůze; začátek schůze + 1h>, tak navrhuji raději delší termín. Předpokládám, že jde o to zamezit nějakému "nepřátelskému aktu" a pokud je interval jen jedna hodina, tak mi to přijde krátké. To raději "začít" schůzi dříve.

Reply:
jde o to, aby "zahájení schůze" netrvalo 5 dní jenom proto, aby se na konci zjistilo, že neni schválenej program... Aktuální návrh řešení (1 hodina) není ideální, ale nic lepšího zatím nepadlo.

Reply:
S tím souhlasím, ale hodinu můžeš strávit i v zácpě, do který jsi neplánoval spadnout. Když to bude 6 hodin, tak už je pravděpodobnost že nestihneš podat zamítnutí podstatně nižší. U té hodiny mi to přijde opravdu hodně pravděpodobný. 
Pravda je, že jsem zatím nevymyslel případ kdy by to mohlo opravdu uškodit i v případě nějakého nepřátelského úmyslu, protože i pokud by někdo protlačil takto nečistě nějaký "špatný" program jednání, tak se prostě neodhlasuje, protože na to je času dost. A pak můžeme řešit co s tím třeba přes KK. 
Nemůže to být nějak zneužitelné naopak ve smyslu, že se něco odhlasuje a někdo tu hodinovou lhůtu bude moct napadnout a prohlašovat pořád dokola, že nemohl zamítnout program a tím že je hlasování neplatné? Prostě takový sociální DoS?

Reply:
Sociální DoS by musel ten dotyčnej řešit soudně. 

Osobně jsem tam dal ten schvalovací default hlavně proto, že minimálně v začátku teď netušíme, jestli to dálkový hlasování vůbec bude fungovat a kdyby se náhodou stalo, že fungovat nebude, byl by spolek kompletně paralyzovanej, protože by fakticky nemohl zahájit schůzi členů.

Souhlas že hodina je málo. Ale když už tam bude 6 hodin, může tam úplně stejně být 24 hodin a schůze se holt de-facto bude svolávat 15 dní předem a ne 14 s tím, že den před zahájením se musí schválit program.

Další možnost by byla řešit schválení programu bodem 7, ale tam neumim říct, jaký by to mělo potenciální legální konsekvence.

Reply:
A proč není možné napsat, že se považuje za schválený pokud do začátku schůze není zamítnutý? Tj. že by bylo přímo ve stanovách, že během těch 14 dnů od odeslání pozvánky do začátku schůze se může program zamítnout a pokud se do začátku nezamítne, tak je považován za schválený. 

BTW co to znamená přesně "zamítnout program"? To znamená, že když libovolný jeden člen napíše, že program zamítá, tak je zamítnutý?

Reply:
Protože schválení programu je součástí schůze, tj. nejdřív se musí ta schůze zahájit, pak se teprve může hlasovat.

Zamítnutí je defaultní stav hlasování ve všech případech, kromě případu, kdy dojde ke schválení (tj. prostá většina přítomných členů hlasuje PRO).

Reply:
fixed by 9213ceb. Limit 24 hodin, implicitní schválení zachováno